esdoc.json

```diff
@@ -38,6 +38,10 @@
       "./doc/tutorial/basics.md",
       "./doc/tutorial/applications.md",
       "./doc/external-references.md"
+    ],
+    "usage": [
+      "./doc/writing-marble-tests.md",
+      "./doc/operator-creation.md"
```

Comment:
@staltz The screen shot is as below: 
![image](https://cloud.githubusercontent.com/assets/6177271/26606216/ab5edcea-45c3-11e7-9321-6b44531d0f4d.png)